src/widgets/splits/Split.cpp

```diff
@@ -498,7 +498,7 @@ void Split::addShortcuts()
          }},
         {"focus",
          [this](std::vector<QString> arguments) -> QString {
```

Comment:
These arguments could be `const std::vector<QString> &`. Maybe we can move them to `std::span<const QString>` some day...